proof/verifier.go

```diff
@@ -252,7 +253,8 @@ func (p *Proof) verifyExclusionProofs() (*commitment.TapCommitmentVersion,
 	}
 
 	// Early pass to determine if we're dealing with v0 or v1 proofs.
-	hasV1Proofs := len(p.ExclusionProofs) > 0 &&
+	hasV1Proofs := p.IsVersionV1() &&
```

Comment:
If this is a V1 proof and there are no STXO proofs, should that result in an error?
Same for the inclusion proofs above.

Reply:
Yes, but there need not necessarily be any exclusion proofs, right? We can have a single output anchor tx that wouldn't necessitate any exclusion proofs. So we can't err out if we don't find them.